genet/__init__.py

```diff
@@ -14,6 +16,10 @@
     read_osm,
 )
 from genet.max_stable_set import MaxStableSet
+from genet.output import geojson
 from genet.schedule_elements import Route, Schedule, Service, Stop
 from genet.use.road_pricing import Toll
-from genet.utils import elevation, google_directions, graph_operations
+from genet.utils import elevation, google_directions, graph_operations, spatial
+
+logging.basicConfig(format="%(levelname)-3s %(asctime)s - %(message)s", level=logging.INFO)
```

Comment:
For logging, what I would suggest is something like this (based on what we do in [calliope](https://github.com/calliope-project/calliope/blob/3779e0dff95c26141293a528f3ec4416cc292783/src/calliope/core/util/logging.py#L19)):

```python
_time_format = "%Y-%m-%d %H:%M:%S"
root_logger = logging.getLogger("genet")  # Get the root logger

# Remove any existing output handlers from root logger
if root_logger.hasHandlers():
    root_logger.handlers.clear()

# Create a console log handler with decent formatting and attach it
formatter = logging.Formatter(
        "%(levelname)-3s %(asctime)s - %(message)s", datefmt=_time_format
)
console = logging.StreamHandler(stream=sys.stdout)
console.setFormatter(formatter)
root_logger.addHandler(console)
```
Then in any file that you want to use a logger, just use `logger = logging.getLogger(__name__)` and because it is a child of the root logger (e.g., `genet.core` is a child of `genet`, as is `genet.cli`) it will inherit the root logger's handler. No need to reimplement reference to the format or anything like that.